content/introduction.html

```diff
@@ -22,23 +25,38 @@ <h1> Preface </h1>
 
 <p class="full">
 The findings in this report were derived from the results of a class survey.
-In total, 113 out of 141 (80%) graduating students responded. SE classes
-can vary significantly depending on the year, so generalize from this survey
-to other classes at your own risk.
+In total, 66 out of 92 (72%) graduating students, and 7 who were in SE but
+transferred out, responded. An addendum survey was also sent out, where
+45 more responses were retrieved, likley from a subset of previous
+respondants.
+  <br /><br />
+
+SE classes can vary significantly depending on the year, so generalize from
+this survey to other classes at your own risk.
   <br /><br />
 
 Study terms are defined as 1A, 1B, 2A, up to 4B (8 terms). 4 month co-op terms
-occur after every study term starting after 1B and ending before 4B.
+occur after every study term starting after 1B and ending before 4B. Government
+shutdowns due to the Covid-19 virus began in March 2020, partway through the 5th
+co-op term. The 4A and 4B study terms were both done remotely due to the pandemic.
+The final coop term was also conducted remotely for most students.
 <br /><br />
 
-All analysis involving income is expressed in Canadian dollars (CAD). The exchange
-rate used to convert from USD to CAD depends on time and is stated where
-relevant. All salaries include monthly housing stipend or the monthly cost of
-corporate housing if applicable.
+All analysis involving income is expressed in Canadian dollars (CAD). A
+constant rate of 1 USD to 1.3 CAD was used to make comparisons consistent and
+simple. As this rate varied over time, 1.3 was chosen due to it being an
```

Comment:
```suggestion
simple. As this rate varied over time, 1.3 was chosen due to its being an
```
[some grammar kink](https://www.merriam-webster.com/words-at-play/breaking-down-possesive-grammar), where the fact that being a constant rate was why it was chosen. but honestly, it's not that important

Reply:
This is actually really cool. I never knew about this, thanks for sharing!